eng/packaging.targets

```diff
@@ -42,4 +51,63 @@
     <MSBuild Projects="$(PkgProjPath)"
              Targets="Build" />
   </Target>
+    
+  <Target Name="AddRefAssemblies" Condition="'$(ExcludeFromPackage)' == '' and '$(TargetsAnyOS)' == 'true'"> 
```

Comment:
We should force this to be opt-in rather than opt-out.  For most packages we should not include reference assemblies.  Also, it looks like this would still run if `'$(ExcludeFromPackage)' == 'false'`

Reply:
I wonder if we can completely omit reference assemblies from packages for the remaining packages now?  I don't think there is any place remaining where we are missing an assembly (was the case in 1.x days when we didn't have PNSE's) nor is there a case where our src assembly has type-forwards by ref does not (was the case in 1.x for things like System.Runtime, up until recently was true for Pipes.AccessControl, but we've dead-ended that).  Can we cut this completely?

Reply:
I added the opt-in feature. I will suggest to merge the infra for the ref assemblies in with this project.
We can create an issue to keep track of deleting all the ref assemblies if we want to go ahead with this. 
would deleting the ref assemblies have any effect on compile times ?